tools/json2dbc.py

```diff
@@ -43,6 +43,8 @@ class Signal:
     offset: float = 0.0
     min: float = 0.0
     max: float = 0.0
+    is_multiplexer: bool = False
+    multiplexer_id: typing.Optional[int] = None
```

Comment:
It's probably better to keep the "multiplexor" spelling for searchability and consistency.